### PR TITLE
Add TimeSlider to SST Map

### PR DESCRIPTION
**Pull request by lboyd93**
This PR is to add a TimeSlider to the MapView in the SST map to better visualize the SST data overtime.

### CODE REVIEW

**Comment by lboyd93** [comment]
@benelan @banuelosj - if you guys want to code review, feel free! Otherwise, I'll merge 😄

**Review by banuelosj — APPROVED** [review]
Looks good to me 👍

File: src/components/Map.tsx [hunk 1]
```diff
@@ -64,7 +68,21 @@ const Map: React.FC = (): JSX.Element => {
...
+      // time slider widget initialization
+      const timeSlider = new TimeSlider({
+        container: "timeSlider",
+        mode: "instant",
+        view,
+        timeVisible: true
+      });
 
+      const legend = new Legend({
+        view: view
```
Comment (benelan) [inline src/components/Map.tsx]: Nitpick: `view: view` can be shortened to `view` like on line 75.

**Review by benelan — APPROVED** [review]
LGTM! "Nitpicks" don't require changes since they are just code smell stuff, but might as well point it out for learning purposes. At least that's how my team does it.

File: src/components/Map.tsx (hunk 1 above)
Comment (lboyd93) [reply src/components/Map.tsx]: Thank you! I keep forgetting

**Pull request merged**